### PR TITLE
mdd-seo-animations

### PR DESCRIPTION
**Pull request by mddifilippo89**
mdd-seo-animations

added new line chart with transitions enabled

### CODE REVIEW

File: samples/charts/category-chart/line-chart-with-animations/App.razor [hunk 1]
```diff
@@ -0,0 +1,46 @@
+@using IgniteUI.Blazor.Controls
+@inject IIgniteUIBlazor IgniteUIBlazor
+
+<div class="igContainer">
+    <div class="igOptions">
+        <span class="igLegend-title">Renewable Electricity Generated</span>
+        <div class="igLegend">
+            <Legend @ref="LegendRef" Orientation="LegendOrientation.Horizontal" />
+        </div>
```
Comment (HUSSAR-mtrela) [inline samples/charts/category-chart/line-chart-with-animations/App.razor:9]: add a combobox to change TransitionIn modes
add a slider to change duration of TransitionIn animation (50ms - 500ms)
add a button to replay animation by forcing the chart to refresh

**Review by HUSSAR-mtrela — CHANGES_REQUESTED** [review]

File: samples/charts/category-chart/line-chart-with-animations/App.razor (hunk 1 above)
Comment (mddifilippo89) [reply samples/charts/category-chart/line-chart-with-animations/App.razor]: pending

File: samples/charts/category-chart/line-chart-with-animations/App.razor [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+            <option>@CategoryTransitionInMode.SweepFromCenter</option>
+            <option>@CategoryTransitionInMode.SweepFromLeft</option>
+            <option>@CategoryTransitionInMode.SweepFromRight</option>
+            <option>@CategoryTransitionInMode.SweepFromTop</option>
+            <option>@CategoryTransitionInMode.SweepFromValueAxisMaximum</option>
+            <option>@CategoryTransitionInMode.SweepFromValueAxisMinimum</option>
+            <option>@CategoryTransitionInMode.Auto</option>
+        </select>
+        <label class="igOptions-value">@IntervalLabel</label>
+        <input class="igOptions-slider" type="range" min="1000" max="8000" step="1000"
```
Comment (HUSSAR-mtrela) [inline samples/charts/category-chart/line-chart-with-animations/App.razor:30]: use min = 50, max = 2000, step = 50

**Review by HUSSAR-mtrela — CHANGES_REQUESTED** [review]

File: samples/charts/category-chart/line-chart-with-animations/App.razor (hunk 2 above)
Comment (mddifilippo89) [reply samples/charts/category-chart/line-chart-with-animations/App.razor:30]: pending

File: samples/charts/category-chart/line-chart-with-animations/App.razor [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+            _transitionInterval = value;
+            ChartKey = new Random().Next().ToString();
+            StateHasChanged();
+        }
+    }
+    private string IntervalLabel
+    {
+        get
+        {
+            return (_transitionInterval / 1000.0).ToString("0.0") + "s ";
```
[the remark this answers is not in the document]
Comment (mddifilippo89) [reply samples/charts/category-chart/line-chart-with-animations/App.razor:74]: pending

**Review by HUSSAR-mtrela — APPROVED** [review]

**Pull request merged**